### ISSUE TITLE
Counter in project template fails to increment, in Chrome (due to auto-translate)

### ISSUE THREAD

**Issue by Venryx** [web-infra-dev/rspack#4377]
### System Info

System: Windows 10
Browsers: Chrome: `Version 117.0.5938.150 (Official Build) (64-bit)`
npmPackages: none, other than the default installed with `npm create rspack@latest`

### Details

I created my first RSPack project, using the command `npm create rspack@latest`, then choosing "react-ts". The files it created all look good, and the page loaded fine.

However, I noticed that the "count is 0" button wouldn't do anything when clicked. I looked at the code and it looked correct. And I added logging, and it was logging the correct count.

I then looked in the dev-tools "Elements" panel, and saw this:
![](https://i.imgur.com/Xu4FbZb.png)

As seen, something is modifying the elements tree to add those "font" elements in-between, breaking the ability for ReactJS to update the contents properly.

I suspected a Chrome extension was causing the problem, so I tried opening the page in Chrome's "Incognito mode". That made me notice the "translation" popup, and that the problem was due to Chrome doing "auto translate" of the page "from Chinese to English" -- and that when it did this "translation", it was modifying the elements tree within the button.

I confirmed this by noticing the icon in the address-bar for the translation, and telling it to "Never translate this site", which avoided the issue:
![](https://i.imgur.com/hSPA4TY.png)

In summary: The fix would seem to be either:
1) Stop Chrome from thinking the page is in Chinese, so it doesn't try to translate the page.
2) Modify how the page is rendered somehow, to make it work even after Google's attempted translation.

I think solution 1 is easier; I think it would be as simple as removing this highlighted text from `index.html`:
![](https://i.imgur.com/LNzRHkS.png)

### Reproduce link

https://github.com/debate-map/web-extension/tree/rspack-issue1

### Reproduce Steps

Steps:
1) Download the linked repo. (or just create the default "react-ts" project using `npm create rspack@latest`)
2) Run: `npm run dev`
3) Open the `localhost:8080` in Chrome, with your language set to English.

Chrome will then try to auto-translate the site, breaking the "count is 0" counter from updating past 0.

**Comment by Venryx** [comment]
Update: I've confirmed that the highlighted text is the cause of the issue.

However there's a couple things to note:
1) The counter getting stuck at 0 "by default" is because apparently I had previously enabled "Always translate Chinese (simplified)"; users who haven't done this probably won't have the issue. (but _will_ still get the annoying translation popup under address-bar, so still worth fixing)
2) Adding/removing that highlighted section apparently is not monitored by HMR; I had to end and rerun the `npm run dev` command for the html language change to be reflected.
